build-system/pr-check/utils.js

```diff
@@ -294,9 +297,19 @@ function uploadBuildOutput(functionName) {
  * @param {string} functionName
  */
 async function uploadDistOutput(functionName) {
+  uploadOutput_(functionName, DIST_OUTPUT_FILE, DIST_OUTPUT_DIRS);
+}
+
+/**
+ * Replaces URLS in HTML files, zips and uploads dist output,
+ * and signals to the AMP PR Deploy bot that the upload is complete.
+ * @param {string} functionName
+ */
+async function processAndUploadDistOutput(functionName) {
   await replaceUrls('test/manual');
   await replaceUrls('examples');
-  uploadOutput_(functionName, DIST_OUTPUT_FILE, DIST_OUTPUT_DIRS);
+  uploadDistOutput(functionName);
```

Comment:
Since `uploadDistOutput()` is an `async` function, does calling it need an `await`?